conf/cubrid.conf

```diff
@@ -76,4 +76,4 @@ log_max_archives=0
 # for deduplicate (-1 ~ 14, -1 forces explicit statements to also be ignored.)
 deduplicate_key_level=-1
 print_index_detail=no
-
+memory_monitoring=n
```

Comment:
```suggestion
memory_monitoring=no
```

Reply:
For consistency with print_index_detail=`no`

Reply:
Is it a spec to set it NO explicitly though it's by default? If it's not, I think we should remove this.

Reply:
Done. I remove it.